packages/maskbook/src/settings/settings.ts

```diff
@@ -52,12 +53,27 @@ export const enableGroupSharingSettings = createGlobalSettings<boolean>('experim
     secondary: () => '(Unstable) Automatically share posts to a group',
 })
 
-export const currentImagePayloadStatus = createNetworkSettings('currentImagePayloadStatus')
-export const currentSelectedIdentity = createNetworkSettings('currentSelectedIdentity')
-export const currentSetupGuideStatus = createNetworkSettings('currentSetupGuideStatus')
+//#region network setting
+
+/**
+ * Expected Usage：export const currentImagePayloadStatus = createNetworkSettings('currentImagePayloadStatus')
+ *
+ * Work around the issue:
+ *      https://github.com/microsoft/TypeScript/issues/42873
+ *      https://github.com/microsoft/TypeScript/issues/30858
+ *
+ * References:
+ *      PluginGitcoinMessages: packages/maskbook/src/plugins/Gitcoin/messages.ts
+ *      PluginTraderMessages: packages/maskbook/src/plugins/Trader/messages.ts
+ *      PluginTransakMessages: packages/maskbook/src/plugins/Transak/messages.ts
```

Comment:
IMO no need to list usages

Reply:
I wanted remind others that these files were changed for this same issue. If need to be removed later, those files should be together.